src/main/java/com/sourcery/gymapp/backend/authentication/mapper/EmailTemplateMapper.java

```diff
@@ -0,0 +1,37 @@
+package com.sourcery.gymapp.backend.authentication.mapper;
+
+import com.sourcery.gymapp.backend.authentication.model.User;
+import com.sourcery.gymapp.backend.events.EmailSendEvent;
+import org.springframework.stereotype.Component;
+
+@Component
+public class EmailTemplateMapper {
+
+    public EmailSendEvent toRegisterVerificationEmail(User user, String verificationUrl) {
+        String subject = "Email verificaton";
+        String senderName = "User Registration Portal Service";
+        String mailContent = "<p> Hi, %s </p>".formatted(user.getUsername()) +
```

Comment:
you can use """ for long Strings, but ofc, it would be even better to store it in application.yaml 

""" Really
long
String """

would be the same as "Really" +
"long" +
"String"

Reply:
I decided to use Text Blocks (triple  """) instead of yaml files, since it will be everything at one place and easier to modify in the future